docs/meshcloud.chargeback-v2.md

```diff
@@ -0,0 +1,82 @@
+
+![Overview](assets/chargeback-v2/overview.png)
+![Details](assets/chargeback-v2/details.png)
+
+- **API and CSV Export Updates**
+
+A new version of the **meshTenantUsageReport API (v3)** will be released, providing additional fields and more detailed responses. This will help make integrations and data handling more efficient.
+
+- **API Enhancements**: More data points, including tags, payment method IDs, tenant IDs, platform type, and more will be available. So meshTenantUsageReport will include all parameters and responses same as for meshChargebacks API and more, so please use meshTenantUsageReport instead for your automation.
+- **CSV Export**: Only the latest data will be included in CSV exports without the previous one for your convenience
+
+> Please note also that moving tenant functionality is available only for those who are on the Chargeback 2.0 version
+> 
+
+## What action do you need to take
+
+### **Update Your Integration**
+
+- **API Update**: If you use the **meshTenantUsageReport API**, switch to the new **v3 API** to take advantage of expanded data fields and enhanced features.
+- **Late Bills via API**: To track changes made to past periods after your last financial reports use **meshTenantUsageReport API by** specifying the **period** and using the **finalizedAfter** parameter to identify any updates that occurred after your last financial reports. For example, if you have done your report on 04.12.2024 by collecting data via API, then you can next time specify the period 2024-12 and finalizedAfter 04.12.2024, so that you will have all data for December that finalized after your report. This approach works for both the cost-forward method, where corrections are applied to future months, and the cost-backward method, where corrections are applied to the period in which the costs were originally incurred.
+
+> Please note that you can switch back to Chargeback 1.0 by navigating to the **Settings** and then the **Configuration** page. However, starting **March 1, 2025**, Chargeback 2.0 will become the default version, and the option to revert to Chargeback 1.0 will no longer be available.
+> 
+
+## Limitations of Chargeback 2.0
+
+While Chargeback 2.0 significantly improves the transparency and accuracy of chargeback statements, there are a few limitations to consider:
+
+### Cost Backward approach in meshstack UI
+
+As mentioned earlier, you can use the API to collect costs that changed after the finalization date and apply corrections to future periods. However, in the **meshStack UI**, we use the cost-backward approach, so chargebacks and TUPRs are updated for the original usage month. Late bills and other corrections are now reflected in the correct month, with their status updated, even after the chargeback statement is finalized.
```

Comment:
d: I think we should stick to the [glossary](https://www.notion.so/meshcloud/Product-Glossary-and-Terminology-00dce982e47644d7a3603532ad438394), so we are consistent with our product terms

```suggestion
As mentioned earlier, you can use the API to collect costs that changed after the finalization date and apply corrections to future periods. However, in the panel, we use the cost-backward approach, so chargebacks and TUPRs are updated for the original usage month. Late bills and other corrections are now reflected in the correct month, with their status updated, even after the chargeback statement is finalized.
```